e2e/tests/components/details/default.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, test } from '../../../fixtures'
+
+test.describe('Detail, default', () => {
```

Comment:
Minor – would be good to make the e2e test names consistent within themselves (even if they're capitalised different from the unit tests :smile: )

```suggestion
test.describe('detail, default', () => {
```